igneous/task_creation.py

```diff
@@ -686,7 +699,20 @@ def create_inference_tasks(task_queue, image_layer_path, convnet_path,
                     mask_mip=mask_mip
                 )
                 task_queue.insert(task)
-    task_queue.wait()
+    task_queue.wait('Uploading InferenceTasks')
+
+    vol = CloudVolume(output_layer_path, mip=output_mip)
```

Comment:
Up to you, but I would recommend adding all parameters to the "method" section. Otherwise it might be difficult to understand what happened later on.

Reply:
I have added all the parameters to provenance file.